### ISSUE TITLE
Does your faith heavily influence your work? In what ways?

### ISSUE THREAD

**Issue by darrenjennings** [kentcdodds/ama#458]
I've been impressed by your work ethic, your humility and your ostensible time management with family and work. Looking for wisdom on work and faith. How does your faith influence your work? in what ways?

From this convo:
https://twitter.com/kentcdodds/status/1026332340266520576

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
